client/extensions/zoninator/state/data-layer/index.js

```diff
@@ -2,15 +2,18 @@
  * Internal dependencies
  */
 import { addHandlers } from 'state/data-layer/extensions-middleware';
+import feeds from './feeds';
 import zones from './zones';
 import debugFactory from 'debug';
 
 const debug = debugFactory( 'zoninator:errors' );
 
 export default function installActionHandlers() {
-	const added = addHandlers( 'zoninator', zones );
+	if ( ! addHandlers( 'zoninator.zones', zones ) ) {
```

Comment:
Using `mergeHandlers()` might be a bit easier - https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/state/data-layer/index.js#L21